### DISCUSSION TITLE
how to yank into the find input box

### DISCUSSION THREAD

**Discussion by akanosenritu** [whitphx/vscode-emacs-mcx#1136] [Q&A]
How am I supposed to yank something into the find input box? Apparently custom keybinds don't work there, so `ctrl-y` is out, I get it, but `ctrl-v` is also set `emacs-mcx.isearchExit`, which closes the input box without doing anything. So my question is, if `ctrl-v` is blocked, what is the supposed way to paste something into the find input box?

**Comment by whitphx** [comment]
I didn't notice the problem as I'm using macOS where `cmd+v` is used for pasting, so I have not taken care of it.
This should probably be a task ticket in the issues.
What OS are you using? Windows or Linux?

**Comment by michaelsilver** [comment]
I'm running into this problem when using `cmd` as the meta key (`emacs-mcx.useMetaPrefixMacCmd`) -- any suggestions? `cmd+v` doesn't work as that's interpreted as `M-v`

**Comment by tshino** [comment]
I found, on Windows, `shift+insert` would work there as expected.
